deployment/src/main/java/io/quarkus/jsch/deployment/JSchBuildTimeConfig.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.jsch.deployment;
+
+import io.quarkus.runtime.annotations.ConfigDocSection;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+
+@ConfigMapping(prefix = "quarkus.jsch")
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public interface JSchBuildTimeConfig {
+
+    /**
+     * The JSch DevService configuration.
+     */
+    @ConfigDocSection
+    JSchDevServiceConfig devservice();
```

Comment:
I just realized we use `devservices` in Quarkus, better rename it too here

Reply:
Also it would be nice if we could move the `JSchDevServiceConfig` to be an inner interface here, as I did in https://github.com/quarkiverse/quarkus-jgit/blob/main/deployment/src/main/java/io/quarkus/jgit/deployment/JGitBuildTimeConfig.java

Reply:
fixed